maxine/python/euler/problem10.py

```diff
@@ -0,0 +1,11 @@
+"The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.
+
+Find the sum of all the primes below two million."
+
+def is_prime(n):
```

Comment:
You shouldn't have a single-letter variable name if the scope of the variable is more than one line or expression.

Reply:
@maxinems this is still not addressed.

Reply:
@anthony-khong done